products/oracle-database.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Oracle Database
+category: db
+iconSlug: oracle
+permalink: /oracle-database
+versionCommand: show version
+releasePolicyLink: https://www.oracle.com/us/assets/lifetime-support-technology-069183.pdf
+releaseDateColumn: true
+activeSupportColumn: Premier Support
+eolColumn: Extended Support
+
+releases:
+
+-   releaseCycle: "21c"
+    lts: false
+    latest: "21.4.2.0.0"
+    support: 2024-04-30
+    eol: false
+    latestReleaseDate: 2022-12-16
+    releaseDate: 2020-12-01
+    link : "https://docs.oracle.com/en/database/oracle/oracle-database/21/whats-new.html" 
+
+-   releaseCycle: "19c"
+    lts: true
+    latest: "19.17.0.0.221018"
+    support: 2027-04-30
+    eol: false
+    latestReleaseDate: 2023-01-13
+    releaseDate: 2019-02-01
+    link: "https://docs.oracle.com/en/database/oracle/oracle-database/19/rnrdm/database-release-notes.pdf"
```

Comment:
HTML links, such as https://docs.oracle.com/en/database/oracle/oracle-database/19/rnrdm/index.html, should be favored IMHO.